.github/workflows/release.yml

```diff
@@ -3,10 +3,27 @@ name: Release
 on:
   release: { types: [published] }
   workflow_dispatch:
-
+    inputs:
+        version:
+          description: 'Version to simulate for deploy'
+          required: true
 jobs:
+  version-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - if: ${{ github.event == workflow_dispatch }}
+        run: |
+          echo "GITHUB_REF=${{ github.event.inputs.version }}" >> $GITHUB_ENV
+      - name: Check tag version matches artifact versions
+        run: |
+          ./bin/bats --version | grep -f "$GITHUB_REF"
+          grep version package.json | grep -f "$GITHUB_REF
+          grep Version contrib/rpm | grep -f "$GITHUB_REF"
```

Comment:
are all these greps supposed to be `-q`? not reading from file, right?

Reply:
No, they are meant to be `-F` for fixed string, to avoid reading the dots as wildcard.